fineract-provider/src/main/java/org/apache/fineract/infrastructure/configuration/service/ExternalServicesConstants.java

```diff
@@ -21,6 +21,7 @@
 import java.util.HashSet;
 import java.util.Set;
 
+@SuppressWarnings({ "AbbreviationAsWordInName" })
```

Comment:
it would be nice to fix this as well.. I'm hoping Checkstyle is OK with `S3_SERVICE_NAME` (because those truly are global constants), but it probably just doesn't like `ExternalservicePropertiesJSONinputParams` and the others like it? Again, just refactor those to be e.g. `ExternalservicePropertiesJsonInputParams`, that's better, and more consistent.